src/Security/Member.php

```diff
@@ -1702,4 +1711,52 @@ public function getHtmlEditorConfigForCMS()
         // If can't find a suitable editor, just default to cms
         return $currentName ? $currentName : 'cms';
     }
+
+    /**
+     * This needs to be public because it's referenced in getMemberPasswordField() and called in ConfirmedPasswordField
```

Comment:
That comment is for PRs, not for the docs. Please use a doc comment that is useful for developers who may choose to call the method instead.

Reply:
Removed